medusa/server/web/config/post_processing.py

```diff
@@ -213,19 +220,13 @@ def isNamingValid(pattern=None, multi=None, abd=False, sports=False, anime_type=
             return 'invalid'
 
     @staticmethod
-    def isRarSupported():
-        """
-        Test Packing Support:
-            - Simulating in memory rar extraction on test.rar file
-        """
-
+    def isRarSupported(unrar_tool):
```

Comment:
This should have been added as a new route in the API since you're just fixing an old one let's just leave it as is. @fernandog maybe once I've got some time again I'll help you move this over to the new API v2 section and we can fix the case issues then.